Yosemite/Yosemite/PointOfSale/POSProductProvider.swift

```diff
@@ -25,9 +40,38 @@ public final class POSProductProvider: POSItemProvider {
         return resultsController
     }()
 
+    public func providePointOfSaleItemsFromNetwork() async throws -> [POSItem] {
+        do {
+            let products = try await productsRemote.loadAllSimpleProductsForPointOfSale(for: siteID)
+            return mapProductsToPOSItems(products: products)
+        }
+    }
+
+    // Maps result to POSProduct, and populate the output with:
+    // - Formatted price based on store's currency settings.
+    // - Product categories, if any.
+    // - Product thumbnail, if any.
+    private func mapProductsToPOSItems(products: [Product]) -> [POSItem] {
+        let currencyFormatter = CurrencyFormatter(currencySettings: currencySettings)
+        return products.map { product in
+            let formattedPrice = currencyFormatter.formatAmount(product.price, with: currencySettings.currencyCode.rawValue) ?? "-"
```

Comment:
Does the code need to be provided, since we already created the formatter with this same set of currency settings?

Reply:
I'm not sure to understand this one, can you clarify?

Reply:
I think you can just do
```suggestion
            let formattedPrice = currencyFormatter.formatAmount(product.price) ?? "-"
```
because you already gave the `currencyFormatter` these `currencySettings`. Take a look at the `formatAmount` implementation and optional params.

Reply:
I see, thanks! Addressed here: https://github.com/woocommerce/woocommerce-ios/pull/13116/commits/4a40286979fd8a63ad3db548da99e21efb890b3f